### ISSUE TITLE
[Online Accounts] Window appear empty

### ISSUE THREAD

**Issue by dawez** [linuxmint/Cinnamon#7231]
```
 * Cinnamon 3.6.7
 * mint 18.3
 * Graphics 
$ inxi -G
Graphics:  Card: Intel Crystal Well Integrated Graphics Controller
Display Server: X.Org 1.18.4 drivers: intel (unloaded: fbdev,vesa) Resolution: 2880x1800@59.99hz
GLX Renderer: Mesa DRI Intel Haswell Mobile GLX Version: 3.0 Mesa 17.2.4
 * 64 bit
 ```

**Issue**
I am able to create an online account however the window appear just empty [see screenshot]. The only option I have in the window is to remove the account. I have seen the same issue also happening on virtual machine using virtual box or parallels on Mint 18.3. I therefore assume that is not hardware/graphic card related. I tried with exchange and google acounts

![image](https://user-images.githubusercontent.com/2534771/35475956-f6f332a8-03a7-11e8-8a67-c0448e74bae7.png)

**Steps to reproduce**
Create a new account in "Online Accounts" application. Double click on the newly created account the window is just empty.

**Expected behaviour**
The properties window should have some control as can be seen in the picture.

![image](https://user-images.githubusercontent.com/2534771/35475952-d37f507c-03a7-11e8-8de6-d50a8dcd5ed7.png)

**Other information**

**Comment by clefebvre** [comment]
This is upstream from us. This component is gnome-online-accounts. Mint is using version 3.18. It should hopefully work better in Mint 19 with gnome-online-account 3.26.

**Comment by dawez** [comment]
Is there any workaround on that in the meantime?

**Comment by gilbertohasnofb** [comment]
@clefebvre Do you know any workaround for this issue? I am also on Mint 18.3 Cinnamon and I am not planning in upgrading to Mint 19 just yet.

**Comment by gilbertohasnofb** [comment]
I've found a workaround for this issue: after adding my Google account to my Online Accounts, I opened GNOME Calendar > Settings > Add > From The Web... and then I pasted the information from the field "Secret address in iCal format" from my Google Calendar.

**Comment by gilbertohasnofb** [comment]
Ok, so I sort of solved this. I found out that installing the package `evolution-data-server-online-accounts` makes the GNOME Calendar app recognize the Google Account set in the Online Accounts app. Now GNOME Calendar can both read and write events, which is great. A couple of problems persist though:

* the options of enabling/disabling access of certain apps to an online account is still not working. This seems to work on Mint 19+. 
* the GNOME Calendar app version available on Mint 18.3, which is 3.20.4, does not have the option of adding reminders to events. I am unsure whether new versions have this feature.
* similarly, events cannot be tagged (work, birthday, etc.)
* there is no option for recurring events (weekly, monthly, daily, etc.)
